### ISSUE TITLE
Can I run the pretrained models on new databases?

### ISSUE THREAD

**Issue by JoaoLages** [salesforce/TabularSemanticParsing#19]
Is it possible to run the pretrained models on new database+question pairs?

**Comment by youssefmellah** [comment]
Hi @JoaoLages 

Any solution for that?

**Comment by JoaoLages** [comment]
Not really. One would need to look deeply into the code. Most of these models preprocess the SPIDER dataset previously and then use another script to train the model on the preprocessed data.
Finding all these things throughout the code is time consuming. At least [RATSQL + GAP](https://github.com/awslabs/gap-text2sql) released a straighforward notebook on how to do that with their code

**Comment by youssefmellah** [comment]
ok @JoaoLages  
Thank you so much
